packages/paste-design-tokens/tokens/global/font-family.yml

```diff
@@ -8,3 +8,11 @@ props:
     value: "{!font-family-10}"
   font-family-code:
     value: "{!font-family-20}"
+  font-family-text-ja:
```

Comment:
Question: in the ticket, Shadi wanted the token to be `fontFamilyTextJA`, but when it is compiled it is `fontFamilyTextJa`. Is this ok? or do I need to update how the tokens are compiled?

Reply:
`fontFamilyTextJa` is fine, there's only so much you can control when you are automating things.